src/main.jsx

```diff
@@ -1,3 +1,5 @@
+import '@arcgis/core/assets/esri/themes/light/main.css';
+import '@utahdts/utah-design-system-header/css';
```

Comment:
**thought**: consider removing this. It's 47kb which is larger than all of plss's tailwind. Its probably better to recreate it all using tailwind to keep the bundle small.

Reply:
I keep going back and forth on this. They have already done a ton of work that we could take advantage of. It's all just plain CSS. But it also gets very sticky very quickly resolving conflicts with other radix/tailwind stuff.

Reply:
yeah. that's why I recommend we own the whole design stack. We would be able to provide them with great feedback on their css but I don't think it's worth the trouble it has already shown to be and will likely continue to be.

Reply:
We can't remove their stylesheet because it's required for the header. If you are worried about CSS bundle size, maybe there's something that can be done to trim down calcites ~350kb? I wonder if there's a minimum version or something. Maybe we'd have to wire up to their SASS to do it.

Reply:
My thoughts on the design system were to implement it with our own css. It seems like a simple enough component that it wouldn't take too much time. Using tailwind and @apply we can reuse the classes.

Application size is important. Esri is another story and I don't think their issues should lower our expectations. I would love to look into the calcite issue with you. I think I did something along those lines in [parcels](https://github.com/agrc/parcels/blob/main/src/styles/basic.scss) a while ago.

Reply:
I've removed the global "utah-design-system" class on the body so the only place that it's applied is on the header.

Reply:
Ok, https://tailwindcss.com/docs/plugins#forms will help with the input element reset now that the header css isn't modifying those.